.changeset/strange-colts-sniff.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-scaffolder-backend-module-rails': minor
```

Comment:
This change should probably be marked as a patch as we are still on version `v0.4.7` - so version `0` - and it's not breaking.

```suggestion
'@backstage/plugin-scaffolder-backend-module-rails': patch
```